### ISSUE TITLE
no 'Access-Control-Allow-Origin' header

### ISSUE THREAD

**Issue by mesterum** [beatnik-ditty/advent-of-node#4]
My deployment is on a remote server that I access through a VSCode tunnel like https://blabla-4300.euw.devtunnels.ms/
My error in the browser is 
`Access to fetch at 'https://blabla-3333.euw.devtunnels.ms/input?year=2022&day=2&custom=false' from origin 'https://blabla-4300.euw.devtunnels.ms' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled`

How to set up the 'Access-Control-Allow-Origin' header on any route on my express-app ?

**Comment by beatnik-ditty** [comment]
All routes should already have CORS enabled, but it looks like it could have needed some additional configuration. Can you checkout the [bugfix/cors_origin](https://github.com/beatnik-ditty/advent-of-node/tree/bugfix/cors_origin) branch and see if that fixes your issue?

**Comment by mesterum** [comment]
I checked your fix but it doesn't help me. I checked with your fix locally, in the same VM but with a there Firefox browser, on localhost and the headers are there:
`
Access-Control-Allow-Origin: http://localhost:4300
Vary: Origin
Access-Control-Allow-Credentials: true
`
I think the VSCode tunnel cuts out the headers

**Comment by mesterum** [comment]
I could expose the port 3333 to not use the tunnel

**Comment by mesterum** [comment]
I think to a solution: Make me on the express-app a route to serve me static files from a subfolder. There I want to put the generated output build of the react-app.

**Comment by beatnik-ditty** [comment]
Good call, probably should have already done that. `nx run react-app:build` by default outputs right alongside the directory that `express-app` runs from in '/dist', so I simply pointed the new route to that.
